lib/install.ps1

```diff
@@ -355,7 +355,17 @@ function args($config, $dir) {
 function run($exe, $arg, $msg, $continue_exit_codes) {
     if($msg) { write-host $msg -nonewline }
     try {
-        $proc = start-process $exe -wait -ea stop -passthru -arg $arg
+        #Check if arguments were provided in the manifest, and pass them to the installer if present
+        if ($arg)
```

Comment:
This came up when I was adding the Visual C++ Redistributable manifests for #755. Luckily, I found some undocumented installation options and it's now a moot point, but I hoped this would eliminate issues in the future. As a demonstration, I've created a "loud" Visual C++ app manifest with no installer arguments at https://raw.githubusercontent.com/MPLew-is/scoop-experimental/master/visual-c-14-loud.json, or as `visual-c-14-loud` in the bucket at https://github.com/MPLew-is/scoop-experimental.

For me, this was failing every time until I added some argument (I added a dummy argument at one point just to see what would happen, and it did work), with PS saying that it couldn't bind arguments. I can get you the exact error text if you're not able to reproduce it.